features/checkout/addressing_order/addressing_order.feature

```diff
@@ -0,0 +1,38 @@
+@checkout_addressing
+Feature: Addressing an order
+    In order to address an order
+    As a Customer
+    I want to be able to fill addressing details
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "PHP T-Shirt" priced at "$19.99"
+        And I am logged in customer
+
+    @ui
+    Scenario: Address an order without different billing address
+        Given I have product "PHP T-Shirt" in the cart
+        And I proceed with the checkout addressing step
+        When I specify the first name as "Jon"
+        And I specify the last name as "Snow"
+        And I specify the street as "Frost Alley"
+        And I choose "France"
+        And I specify the city as "Ankh Morpork"
```

Comment:
Its **Jon Snow** ordering a **PHP T-Shirt** to **Frost Alley** which is in **Ankh Morpork** in **France**? What a mix! 😲 😄